backend/infrahub/graphql/mutations/ipam.py

```diff
@@ -107,16 +133,38 @@ async def mutate_create(
         namespace_id = await validate_namespace(db=db, data=data)
 
         async with db.start_transaction() as dbt:
-            address = await cls.mutate_create_object(data=data, db=dbt, branch=branch)
-            reconciler = IpamReconciler(db=dbt, branch=branch)
-            reconciled_address = await reconciler.reconcile(
-                ip_value=ip_address, namespace=namespace_id, node_uuid=address.get_id()
-            )
-
-        result = await cls.mutate_create_to_graphql(info=info, db=db, obj=reconciled_address)
+            lock_name = cls._get_lock_name(namespace_id, branch)
+            if lock_name is not None:
```

Comment:
you could use a walrus operator here 
```suggestion
            if lock_name := cls._get_lock_name(namespace_id, branch)
```